core/trino-main/src/test/java/io/trino/server/security/oauth2/TestOAuth2WebUiAuthenticationFilterWithJwt.java

```diff
@@ -77,4 +94,44 @@ protected void validateAccessToken(String cookieValue)
         assertThat(claims.get("client_id")).isEqualTo(TRINO_CLIENT_ID);
         assertThat(claims.getIssuer()).isEqualTo("https://localhost:4444/");
     }
+
+    @Test
+    public void testLowercaseTrinoUserWillNotTriggerImpersonationWithAppropriateUserMapping()
+            throws Exception
+    {
+        String usernameUppercase = TRINO_CLIENT_ID.toUpperCase(ENGLISH);
+        String usernameLowerCase = TRINO_CLIENT_ID.toLowerCase(ENGLISH);
+        assertThat(usernameUppercase).isNotEqualTo(TRINO_CLIENT_ID);
+        assertThat(usernameLowerCase).isNotEqualTo(TRINO_CLIENT_ID);
```

Comment:
It's lower-case atm:
```
2021-11-17T16:03:35.0187187Z [ERROR]   TestOAuth2WebUiAuthenticationFilterWithJwt.testLowercaseTrinoUserWillNotTriggerImpersonationWithAppropriateUserMapping:105 
2021-11-17T16:03:35.0190793Z Expecting:
2021-11-17T16:03:35.0191552Z  <"trino-client">
2021-11-17T16:03:35.0191930Z not to be equal to:
2021-11-17T16:03:35.0192415Z  <"trino-client">
```

Reply:
👍 Thx I accidentally removed mixed case client id change I've made earlier.